.github/workflows/build.yml

```diff
@@ -366,7 +375,7 @@ jobs:
           bash -c "
             cat src/palace/manager/_version.py &&
             source env/bin/activate &&
-            poetry install --without ci --no-root --sync &&
+            poetry sync --without ci --no-root &&
```

Comment:
Latest poetry has deprecated the `poetry install --sync` command in favor of `poetry sync`, so I switched to using it here.